.travis.yml

```diff
@@ -25,17 +25,17 @@ install:
 - yarn config list
 - yarn
 script:
-- ./ci-script.sh
+- ./tasks/ci-script.sh
 after_success:
-- chmod ugo+x ./deploy.sh
-- "./deploy.sh"
+- ./tasks/deploy.sh
+- if $RELEASE; then yarn run semantic-release; fi
 after_failure:
 - "./cypress/support/s3-artifact-upload.sh"
 branches:
   except:
   - "/^v\\d+\\.\\d+\\.\\d+$/"
 env:
   global:
-  - S3SCREENSHOT=true
-  - S3COMPARISON=true
-  - S3VIDEO=true
+    S3SCREENSHOT: true
+    S3COMPARISON: true
+    S3VIDEO: true
```

Comment:
These were changed because the API config merge doesn't play as nicely with the list syntax version.